Sources/BraintreeCard/BTCardNonce.m

```diff
@@ -74,7 +74,7 @@ + (NSString *)typeStringFromCardNetwork:(BTCardNetwork)cardNetwork {
             return @"Solo";
         case BTCardNetworkSwitch:
             return @"Switch";
-        case BTCardNetworkUKMaestro:
+        case BTCardNetworkUkMaestro:
```

Comment:
When converting the `BTCardNetwork` enum to Swift there is not a way to give the Obj-C case a different name than the Swift case. Due to how Obj-C maps enum names to Swift this worked and gave us `.ukMaestro` in Swift. there is not an equivalent to this mapping in Swift so this case name changed and was added to the CHANGELOG.